config/settings_schema.json

```diff
@@ -133,6 +133,63 @@
       }
     ]
   },
+  {
+    "name": "Grid",
+    "settings": [
+      {
+        "type": "paragraph",
+        "content": "Affects collection, search and blog pages. Also affects collage, collection, collection list, multicolumn and blog sections"
+      },
+      {
+        "type": "header",
+        "content": "Desktop"
+      },
+      {
```

Comment:
Personally, I feel that placing page-width under grid does not make sense, because page width affects the entire theme, while grid affects only specific sections. 
I feel that Grid is part of the layout instead of Layout being part of grid: Layout > grid.

Layout 
  > Grid 
  > Page width

Reply:
Oh interesting, I didn't see it this way! I am aligned to do the opposite! Thanks!

Reply:
Agreed - the grid is used in specific sections, where `page-width` is used everywhere.